src/pages/thamizarasu.astro

```diff
@@ -0,0 +1,25 @@
+---
+import BioLayout from '@layouts/BioLayout.astro';
+---
+
+<BioLayout
+  name="Thamizarasu Sankara"
+  role="Doctor Legend.jpg"
+  imageAlt="Thamizarasu Sankara's default profile image"
+>
```

Comment:
`role` should be your role, which is "Full Stack Engineer/Scrum Master" according to the home page I wrote.

Also, `image` should be "Doctor Legend.jpg". I prefer no spaces in the filename, so if you can rename the file to have no spaces as well that would be great.

```suggestion
  role="Full Stack Engineer/Scrum Master"
  image="DoctorLegend.jpg"
  imageAlt="Thamizarasu Sankara's default profile image"
>
```